app/assets/javascripts/omni-nav-v2.js

```diff
@@ -105,9 +106,21 @@ var OmniNav2 = (function() {
 
     var syncLinkWidths = function() {
       var width = $('#js-off-canvas-nav > ul').width();
-      offCanvasLinks.css('width', width);
+      $offCanvasLinks.css('width', width);
     };
 
+    var enableMenusToggle = function() {
+      // Enables toggle to slide main/umbrella menus back and forth.
+      $('a.toggle-menu-label').on('click', function(e) {
+        console.debug('toggle menus now!');
+        // Toggles headers.
+        $('div#umbrella-logo').toggle('blind');
+
+        // Slide-toggles the menus.
+        $('div#off-canvas-umbrella').toggle('slide');
+      });
```

Comment:
Notice I'm using jQueryUI API here. (At least, I think I am.) Old OmniNav wasn't. It was a mess.